fleetshard/pkg/central/reconciler/reconciler.go

```diff
@@ -1857,13 +1860,17 @@ func (r *CentralReconciler) ensureRouteDeleted(ctx context.Context, routeSupplie
 }
 
 func getTenantLabels(c private.ManagedCentral) map[string]string {
-	return map[string]string{
+	labels := map[string]string{
 		managedByLabelKey:    labelManagedByFleetshardValue,
 		instanceLabelKey:     c.Metadata.Name,
 		orgIDLabelKey:        c.Spec.Auth.OwnerOrgId,
 		TenantIDLabelKey:     c.Id,
 		instanceTypeLabelKey: c.Spec.InstanceType,
 	}
+	if c.Metadata.Internal {
+		labels[ProbeLabelKey] = "true"
```

Comment:
Since we're matching on `ProbeLabelKey exists`, we don't need to set an actual value to the label. Only the presence of the key matters here

```suggestion
		labels[ProbeLabelKey] = ""
```